SQL-Query/Query-VI./SQLAnswer.md

```diff
@@ -0,0 +1,44 @@
+**6. Total $ value of shipments shipped from facility 904/906 to first quarter:**
+```sql
+select 
+  sum(UNIT_PRICE * QUANTITY) TOTALMONETARYVALUE 
+from 
+  order_item 
+where 
+  ORDER_ID in (
```

Comment:
Can you check writing this query using joins.
And also share their query costs.